flujo/builtins_extras.py

```
@@ -66,6 +81,51 @@ async def render_jinja_template(template: str, variables: JSONObject | None = No
+    sandbox = _resolve_sandbox(context)
+    request = SandboxExecution(
+        code=code,
+        language=language,
+        files=files,
+        environment=environment,
+        arguments=tuple(arguments or ()),
+        timeout_s=timeout_s,
+    )
+    try:
+        result = await sandbox.exec_code(request)
+    except Exception as exc:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)